src/hooks/test/use-popover-should-close-test.js

```diff
@@ -0,0 +1,52 @@
+import { mount } from 'enzyme';
+import { useRef } from 'preact/hooks';
+
+import usePopoverShouldClose, { $imports } from '../use-popover-should-close';
+
+describe('usePopoverShouldClose', () => {
+  function FakeComponent() {
+    const popoverEl = useRef();
+
+    usePopoverShouldClose(popoverEl, () => {});
+
+    return <div ref={popoverEl} />;
+  }
+
+  let fakeUseClickAway;
+  let fakeUseFocusAway;
+  let fakeUseKeyPress;
+
+  beforeEach(() => {
+    fakeUseClickAway = sinon.stub();
+    fakeUseFocusAway = sinon.stub();
+    fakeUseKeyPress = sinon.stub();
+
+    $imports.$mock({
+      './use-click-away': {
+        useClickAway: fakeUseClickAway,
+      },
+      './use-focus-away': {
+        useFocusAway: fakeUseFocusAway,
+      },
+      './use-key-press': {
+        useKeyPress: fakeUseKeyPress,
+      },
+    });
+  });
+
+  afterEach(() => {
+    $imports.$restore();
+  });
+
+  function createElement() {
+    return mount(<FakeComponent />);
+  }
+
+  it('combines useClickAway, useFocusAway and useKeyPress', () => {
+    createElement();
+
+    assert.called(fakeUseClickAway);
+    assert.called(fakeUseFocusAway);
+    assert.called(fakeUseKeyPress);
```

Comment:
Just mocking the other hooks and verifying they are called, as those hooks have their own tests.